package.json

```diff
@@ -25,11 +25,11 @@
   "license": "Apache-2.0",
   "scripts": {
     "build": "rm -rf dist && mkdir -p dist/public && npm run build:assets && npm run build:html && npm run build:img && npm run build:ico",
-    "build:assets": "NODE_ENV=production webpack -p",
+    "build:assets": "NODE_ENV=production webpack -p --display-optimization-bailout",
     "build:html": "cp public/*.html dist/.",
     "build:ico": "cp public/*.ico dist/.",
     "build:img": "cp public/*.png dist/public/.",
-    "start": "NODE_ENV=development webpack-dev-server --config webpack.config.js --progress --colors",
+    "start": "NODE_ENV=development webpack-dev-server --progress --colors --display-optimization-bailout",
```

Comment:
Ah, I didn't actually test that... It can be removed, it's really just useful for production